onyo/lib/commands.py

```diff
@@ -137,71 +139,76 @@ def onyo_cat(inventory: Inventory,
              paths: list[Path]) -> None:
     r"""Print the contents of assets.
 
-    At least one valid asset path is required.
-    The same paths can be given multiple times.
-    If any path specified is invalid, no contents are printed and an error is raised.
+    The same path can be given multiple times.
+
+    If any path is not an asset, nothing is printed.
+    If any asset content is invalid, the content of all assets is still printed.
 
     Parameters
     ----------
     inventory
         The inventory containing the assets to print.
     paths
-        Path(s) to assets for which to print the contents.
+        Path(s) of assets to print the contents of.
```

Comment:
Again, something I am not sure about, but:
`Path(s)` is the way we wrote it in the beginning. Now we often just talk about "assets", "directories", and "paths" in doc-strings (when they are not help-text?).
So while reading this PR it felt a bit like a "surprising" inconsistency here, even though a small one, obviously :P

Reply:
Thanks for pointing this out. I've dropped the `(s)` to make it match everything else.